<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-A64RUOI" ignored="1" name="Fourth Wave" tests="20" failures="1" timestamp="13 May 2017 21:58:28 GMT" time="1.633" errors="0">
  <testcase name="@BeforeMethod setUp" time="0.007" classname="il.co.topq.difido.DifidoReporterTestsPart1">
    <failure type="java.lang.Exception" message="My failure at before method">
      <![CDATA[java.lang.Exception: My failure at before method
	at il.co.topq.difido.DifidoReporterTestsPart1.setUp(DifidoReporterTestsPart1.java:40)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod setUp -->
  <testcase name="@AfterMethod tearDown" time="0.0" classname="il.co.topq.difido.DifidoReporterTestsPart1">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="@AfterMethod tearDown" time="0.0" classname="il.co.topq.difido.DifidoReporterTestsPart1">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="@AfterMethod tearDown" time="0.0" classname="il.co.topq.difido.DifidoReporterTestsPart1">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="@AfterMethod tearDown" time="0.0" classname="il.co.topq.difido.DifidoReporterTestsPart1">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="@AfterMethod tearDown" time="0.0" classname="il.co.topq.difido.DifidoReporterTestsPart1">
    <skipped/>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="arg0  null" time="0.002" classname="il.co.topq.difido.DifidoReporterTestsPart1">
    <failure type="java.lang.Exception" message="This is my failure">
      <![CDATA[java.lang.Exception: This is my failure
	at il.co.topq.difido.DifidoReporterTestsPart1.testWithFailure(DifidoReporterTestsPart1.java:97)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 9 stack frames]]>
    </failure>
  </testcase> <!-- arg0  null -->
  <testcase name="arg0  arg1" time="0.002" classname="il.co.topq.difido.DifidoReporterTestsPart1"/>
  <testcase name="arg0  arg1" time="0.001" classname="il.co.topq.difido.DifidoReporterTestsPart1"/>
  <testcase name="arg2  arg3" time="0.002" classname="il.co.topq.difido.DifidoReporterTestsPart1"/>
  <testcase name="arg6  arg7" time="0.001" classname="il.co.topq.difido.DifidoReporterTestsPart1"/>
  <testcase name="arg4  arg5" time="0.001" classname="il.co.topq.difido.DifidoReporterTestsPart1"/>
  <testcase name="arg8  arg9" time="0.003" classname="il.co.topq.difido.DifidoReporterTestsPart1"/>
  <testcase name="failOnBeforeMethod  null" time="0.0" classname="il.co.topq.difido.DifidoReporterTestsPart1">
    <skipped/>
  </testcase> <!-- failOnBeforeMethod  null -->
  <testcase name="null  null" time="0.002" classname="il.co.topq.difido.DifidoReporterTestsPart1"/>
  <testcase name="arg10  arg11" time="0.001" classname="il.co.topq.difido.DifidoReporterTestsPart1"/>
  <testcase name="arg0  null" time="0.002" classname="il.co.topq.difido.DifidoReporterTestsPart1"/>
  <testcase name="arg12  arg13" time="0.002" classname="il.co.topq.difido.DifidoReporterTestsPart1"/>
  <testcase name="arg0  arg1" time="0.004" classname="il.co.topq.difido.DifidoReporterTestsPart2"/>
  <testcase name="arg0  arg1" time="0.001" classname="il.co.topq.difido.DifidoReporterTestsPart2"/>
  <testcase name="arg0  null" time="0.001" classname="il.co.topq.difido.DifidoReporterTestsPart2"/>
  <testcase name="arg0  arg1" time="0.006" classname="il.co.topq.difido.DifidoReporterTestsPart2"/>
  <testcase name="null  null" time="0.0" classname="il.co.topq.difido.DifidoReporterTestsPart2">
    <skipped/>
  </testcase> <!-- null  null -->
  <testcase name="arg0  arg1" time="0.002" classname="il.co.topq.difido.DifidoReporterTestsPart2"/>
  <testcase name="arg0  arg1" time="1.001" classname="il.co.topq.difido.DifidoReporterTestsPart1"/>
  <testcase name="arg0  arg1" time="1.465" classname="il.co.topq.difido.DifidoReporterTestsPart2"/>
  <testcase name="@BeforeSuite suitePreConfigure" time="0.0" classname="infra.parallel.BaseTest">
    <ignored/>
  </testcase> <!-- @BeforeSuite suitePreConfigure -->
</testsuite> <!-- Fourth Wave -->
